src/shared/views/Login.js

```diff
@@ -213,6 +213,7 @@ class _Login extends React.Component<Props, State> {
                     groupContributionCount: 0,
                     projectContributionCount: 0,
                     taskContributionCount: 0,
+                    username,
```

Comment:
yeah, finally I get the stats in the leaderboard. :D

Reply:
ace!